diff --git a/node_modules/react-native/index.js b/node_modules/react-native/index.js
index c9eb445..dae38cc 100644
--- a/node_modules/react-native/index.js
+++ b/node_modules/react-native/index.js
@@ -314,12 +314,6 @@ module.exports = {
         "It can now be installed and imported from '@react-native-async-storage/async-storage' instead of 'react-native'. " +
         'See https://github.com/react-native-async-storage/async-storage',
     );
-    if (CheckPlatform.isTVOS) {
-      warnOnce(
-        'async-storage-tvos',
-        'Persistent storage is not supported on tvOS, your data may be removed at any point.',
-      );
-    }
     return require('./Libraries/Storage/AsyncStorage');
   },
   get BackHandler(): BackHandler {
